packages/contracts/package.json

```diff
@@ -5,6 +5,7 @@
     "type": "git",
     "url": "https://github.com/aragon/admin-plugin"
   },
+  "version": "1.3",
```

Comment:
Shouldn't this be 1.3.0 for compatibility with the docs?

Reply:
for compatibility it should be 1.3